alws/crud/build_node.py

```diff
@@ -162,6 +216,14 @@ async def __process_logs(pulp_client: PulpClient, task_id: int,
                     href=href
                 )
             )
+    hrefs = [item[0] for item in results]
```

Comment:
There is no need to use list comprehension for retrieving hrefs, above you iterate on results of async tasks and you can append hrefs in some list